src/utils/components/HelpTextIcon/HelpTextIcon.tsx

```diff
@@ -20,7 +20,7 @@ const HelpTextIcon: FC<HelpTextIconProps> = ({
   helpIconClassName = '',
 }) => (
   <Popover aria-label={'Help'} bodyContent={bodyContent} className={className} position={position}>
-    <HelpIcon className={helpIconClassName} />
```

Comment:
I'd suggest to change it to something like this to improve readability
```
import classNames from 'classnames';
...
<HelpIcon className={classNames('help-icon__cursor', helpIconClassName)} />
```

Reply:
Done!